### DISCUSSION TITLE
Cellebrite ufdr Media Classification parser

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#684] [General]
Hi!!

Is there a way to configure iped to keep the media classification made by the Cellebrite Physical Analyzer engine?

**Comment by lfcnassif** [comment]
This is in the current 109 tasks queue: https://github.com/sepinf-inc/IPED/issues/493

Contributions are welcome!

**Reply by gisms** [reply]
Yeap, thanks

**Reply by lfcnassif** [reply]
Just to let you know, this was added some while back #493 by @aberenguel

**Reply by gisms** [reply]
Very Nice!!!!!!!!
I'm in a very busy phase, just struggling to cope with the routine, so I couldn't even perform all the tests I'd like on that other whatsapp question.
